files/standalone.xml

```diff
@@ -0,0 +1,349 @@
+<?xml version='1.0' encoding='UTF-8'?> 
```

Comment:
Does the addition of this mean that we now have three versions of standalone.xml to maintain?
- The original file dropped by installing torquebox
- This file
- The standalone.xml.erb dropped by the `pe_razor` module